perfmetrics/scripts/utils/metrics_util_test.py

```diff
@@ -0,0 +1,34 @@
+"""Test for fio output logging."""
+
+import unittest
+import os
+from metrics_util import remove_old_log_files
+
+
+class TestFioLogger(unittest.TestCase):
```

Comment:
update the class and method name as per changes made in metrics_util.py

Reply:
Done